test/sagemaker_tests/pytorch/inference/integration/sagemaker/test_mnist.py

```diff
@@ -118,3 +124,50 @@ def _test_mnist_distributed(
         output = predictor.predict(data)
 
         assert output.shape == (batch_size, 10)
+
+        #  Check for Cloudwatch logs
+        if verify_logs:
+            _check_for_cloudwatch_logs(endpoint_name)
+
+def _check_for_cloudwatch_logs(endpoint_name):
+    client=boto3.client('logs')
+    identifyLogStream = client.describe_log_streams(
+        logGroupName='/aws/sagemaker/Endpoints/'+endpoint_name,    
+        orderBy='LastEventTime',
+        descending=True,
+        limit=5
+    )
+
+    indexError=False;
+    try:
+        logStreamName=identifyLogStream['logStreams'][0]['logStreamName']        
+    except IndexError:
+        indexError=True    
+
+    if(indexError):
+        raise Exception('Exception: Trying to lookup log streams from the given log group failed')
+
+    log_events_response = client.get_log_events(
+        logGroupName='/aws/sagemaker/Endpoints/'+endpoint_name,
```

Comment:
you probably can save log_group_name as a variable since it will likely be useful for error messages as well

Reply:
Done